README.md

```diff
@@ -97,3 +97,5 @@ If you don't want to compile POGOProtos but instead use it directly, check out t
 | Go               | https://github.com/pkmngo-odi/pogo-protos               |
 | Haskell          | https://github.com/relrod/pokemon-go-protobuf-types     |
 | Rust             | https://github.com/rockneurotiko/pokemon-go-protobuf-rs |
+| Java             | https://github.com/celandro/pogoprotos-java             |
+| JSON             | https://github.com/celandro/pokemongo-game-master       | 
```

Comment:
Why use your fork and not [the source](https://github.com/BrunnerLivio/pokemongo-game-master)?